functions/defaulted_parameters.md

```diff
@@ -0,0 +1,87 @@
+# Module name: topic name
+_Skeleton instructions are typeset in italic text._
+
+## Overview
+
+Functions in C++ may be overloaded with different numbers and types of parameters. It may be of value to specify default values for some number of parameters, to allow a consumer to avoid specifying parameters that rarely change, or to enable expanding the set of parameters while maintaining backward compatibility with existing consumers.
+
+<table>
+  <thead>
+    <th>Level</th>
+    <th>Objectives</th>
+  </thead>
+  <tr>
+    <td>Foundational knowledge</td>
+    <td>can make calls to functions with or without defaulted parameters</td>
+  </tr>
+  <tr>
+    <td>Advanced</td>
+    <td>declares and defines functions with defaulted parameters</td>
+  </tr>
+  <tr>
+    <td>Further studies</td>
+    <td>refinement of defaulted parameters through multiple declarations</td>
+  </tr>
+</table>
+
+## Motivation
+
+Default parameters allow the omission of parameters with obvious or common values. Also may be utilized to extend an existing function signature without forcing changes to existing calling code.
+
+## Topic introduction
+
+Explain how default parameters work and how to define them.
+
+## Foundational knowledge: Using *
+
+### Background/Required Knowledge
+
+A student is able to:
+
+* Make calls to existing functions, passing parameters
+* Declare member and non-member functions, separate from definitions
+* Define member and non-member functions
+* Explain what a default constructor is and does
+
+### Student outcomes
+
+A student should be able to:
+
+1. Call to a function with a default parameter with or without that parameter specified
+2. Explain what the requirements are for a type to be usable with a default parameter
+3. Explain when the lifetime of a defaulted parameter begins and ends
```

Comment:
Are you thinking here of something like `void foo(const std::string& s = "hello world")`? Either way, I think you should be. :)  C++ treats the default argument value here _differently_ from how Python famously treats it.

However, note that this is a caveat about _defining_ functions, not _calling_ them.

Reply:
Yes to part A, Not sure I understand your point about defining vs calling functions. Is there a suggested change to the wording there?

Reply:
> Not sure I understand your point about defining vs calling functions

I just meant that the lifetime of `std::string("hello world")` there doesn't matter at all to the _caller_ of the function, who may not even be aware that the default argument exists. If there's a bug in that code, it's a bug written-by and fixable-only-by the person who _defined_ the function with default arguments.  (This was relevant because at this point in the doc we're still in the "Basic" section that purports to have nothing to do with defining functions, only calling them. But probably that should be fixed by moving everything into "Basic".)

Reply:
Just gonna move it all to basic. :-)

Reply:
done